app/views/classified_listings/_form.html.erb

```diff
@@ -45,7 +45,9 @@
         <% end %>
       </div>
     </div>
-      <%= javascript_pack_tag "listingForm", defer: true %>
+      <% unless @organizations.present? %>
```

Comment:
To avoid the issue mentioned above about chunks loading multiple times, I have to check if the user is logged on or not. If they aren't, just load the `"listingForm"` pack. Otherwise, load the `"listingForm"` and `"orgCreditSelector"` packs within the same `javascript_packs_with_chunks_tag` call.

Reply:
Ooops, I meant `@organizations`. 🙃